### PR TITLE
Add reset to PostHog integration

### PR DESCRIPTION
**Pull request by timgl**
## Description of the change

> Add reset() functionality to the posthog integration. Not sure how I can add tests to this?

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

---
This change is [Reviewable](https://reviewable.io/reviews/rudderlabs/rudder-sdk-js/271)

### CODE REVIEW

**Comment by samwinslow** [comment]
Hi there, just following up on this as we have some customers wondering why their data is not being properly ingested into PostHog via RudderStack.

This PR should solve an issue where multiple users that log onto the same device are counted as one, which adversely affects the quality of data emitted by RudderStack and ingested into PostHog.

Please let me know if there's any additional context needed from us — we just want to make this pipeline as standard & predictable as possible.

File: integrations/Posthog/browser.js [hunk 1]
```diff
@@ -115,6 +115,17 @@ class Posthog {
     posthog.capture(event, properties);
   }
 
+  reset() {
```
Comment (saikumarrs) [inline integrations/Posthog/browser.js:118]: Where do you intend to call this method from?

Comment (saikumarrs) [inline integrations/Posthog/browser.js:118]: Where do you intend to call this method from?

**Comment by saikumarrs** [comment]
@timgl @samwinslow This PR has been stale for many days. If you wish to keep it open, please try to respond within the next 10 days.

**Comment by saikumarrs** [comment]
Closing the PR as there is no response from the author.

**Pull request closed without merging**